platform/error/platform-error-impl/package.json

```diff
@@ -44,7 +38,7 @@
     "node-fs": "0.1.7",
     "node-options": "0.0.3",
     "path": "0.4.9",
-    "casperjs": "~1.1.0",
+    "casperjs": "^1.1.0-beta3",
```

Comment:
My preference would be to do (c) but for this ticket/PR you can just do (a) and then create a ticket to do (c) if you prefer.